app/src/main/java/com/example/poste/activities/EditPostActivity.java

```diff
@@ -59,84 +39,73 @@ public class EditPostActivity extends AppCompatActivity {
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_edit_post);
 
-        // Configure window settings for fullscreen mode
-        requestWindowFeature(Window.FEATURE_NO_TITLE);
-        this.getWindow().setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN,
-                WindowManager.LayoutParams.FLAG_FULLSCREEN);
-        getSupportActionBar().hide();
+        DebugUtils.logUserFoldersAndPosts(User.getUser());
 
-        // Set the activity layout
-        setContentView(R.layout.activity_edit_post);
+        EditText linkToPost = findViewById(R.id.editTextLinkToPost);
+        EditText postTitle = findViewById(R.id.editTextPostTitle);
+        EditText postDescription = findViewById(R.id.editTextPostDescription);
+
```

Comment:
I noticed that we don't have editTextPostTags hooked up to anything; should we? What's the desired functionality look like? Should we add an endpoint and include this in API calls? Deferring to you for your take on this

Reply:
So the Tags was there from the prototype. I made the layout following the proto, and then during implementation I found tags wasnt implemented anywhere else... So I left it hanging for now. We can definitely incorporate it now, or after MVP when we begin to implement additional features such as searching for folders by name or posts by title, description or tags. It is based on a brief conversation in chat we had regarding adding searchable tags in the program. The tags could be user input, but my thought was it would probably be an ML classification algo. We can discuss more?

Reply:
I see. Just curious. No worries.